tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -641,6 +641,16 @@ func addStoreToSchedulerConfig(cmd *cobra.Command, schedulerName string, args []
 		cmd.Println(cmd.UsageString())
 		return
 	}
+
+	exist, err := checkSchedulerExist(cmd, schedulerName)
+	if err != nil {
+		return
+	}
+	if !exist {
+		cmd.Printf("Failed! scheduler %s not found.\n", schedulerName)
```

Comment:
It is better to discuss this with PM.

```suggestion
		cmd.Printf("Can't update config for a non-existent scheduler.\n", schedulerName)
```

Reply:
changed to `Unable to update config: schedulerXX does not exist.` PTAL, thx~